.eslintrc.js

```diff
@@ -14,7 +14,9 @@ module.exports = {
   ],
   globals: {
     Atomics: 'readonly',
-    SharedArrayBuffer: 'readonly'
+    SharedArrayBuffer: 'readonly',
+    fixture: true,
+    test: true
```

Comment:
We could add [eslint-plugin-testcafe](https://www.npmjs.com/package/eslint-plugin-testcafe) for this

Reply:
Yeah all it does is exactly these two lines so didn't bother.

Reply:
Just means if they update their API then it will stay in line